### PR TITLE
added config example for using with master key.

### PR DESCRIPTION
**Pull request by otmezger**

### CODE REVIEW

File: _includes/js/getting-started.md [hunk 1]
```diff
@@ -46,6 +46,16 @@ Parse.initialize("YOUR_APP_ID", "YOUR_JAVASCRIPT_KEY");
 Parse.serverURL = 'http://YOUR_PARSE_SERVER:1337/parse'
 ```
 
+If the Masterkey needs to be provided, use this
```
Comment (flovilmart) [inline _includes/js/getting-started.md]: can you note that the master should only be used in safe environments and never be used on the clientside?

**Review by flovilmart — CHANGES_REQUESTED** [review]

File: _includes/js/getting-started.md (hunk 1 above)
Comment (otmezger) [reply _includes/js/getting-started.md]: sure, will try to add it.

Comment (flovilmart) [reply _includes/js/getting-started.md]: Thanks a lot, each time we talk about the master key, we should have a big warning ⚠️ or ‼️ To make sure everyone pays attention :)

**Comment by otmezger** [comment]
⚠️ ok, I made the changes and even put the emojis in place :-)

**Comment by flovilmart** [comment]
That’s awesome thanks! Would be awesome to reuse that snippet everywhere :p

**Pull request merged**